lib/chef/dsl/powershell.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Author:: Jay Mundrawala (<jdm@chef.io>)
+# Copyright:: Copyright (c) 2015 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'chef/util/powershell/ps_credential'
+
+class Chef
+  module DSL
+    module Powershell
+      def ps_credential(username='placeholder', password)
```

Comment:
@smurawski right so, PSCredential.new is going to require a username just because it's a required argument, so other folks using this functionality will have to specify something and think/be aware of that. so that's fine I think. and in this case here we're saying we don't need one, so the most we would do is add a comment here about how DSC ignores the username?